data/hai/unfettered side missions.txt

```diff
@@ -160,267 +96,413 @@ mission "Hai Legacy 1"
+			choice
+				`	(Go back to the shipyard.)`
+			`	They walk away far enough so that you cannot hear them, leaving you with the access codes to the ship so you can bring it to the shipyard. With the aid of a surprisingly advanced navigation AI, you manage to bring it all the way there effortlessly despite the ship's multiple crew design. When you arrive, however, the AI shuts down without explanation, and you're forced to use a full crew complement.`
+			`	You remember the mysterious pair requested you bring only the Anomalocaris. It may be time to sell your other ships, or you could always park them for later.`
 				accept
-			label dumb
+			label foolish
 			`	You get your weapon out and proceed to shoot on the ship.`
+			branch miss
+				random < 50
 			branch lucky?
 				random < 2
 			branch lucky??
-				random < 40
-			`	You shoot the ship, but the small weapon has no chance of doing anything to the heavy armor, even in its current state. The ship's speakers then activate, almost deafening you. "MY TURN!"`
-			`	The ship then cuts you down with surgical accuracy using its Bullfrog Anti-Missiles. You can still hear the two Hai speaking to each other in a surprised tone. "How did this one survive so lon..." And everything goes dark.`
-				die
+				random < 30
+			`	The shot reaches the ship, but the small weapon has no chance of doing anything to the heavy armor, even in its current state. The ship's speakers then activate, almost deafening you. "MY TURN!"`
+			`	The ship then hits your weapon, disarming you with surgical accuracy using its Bullfrog Anti-Missiles. One of the two Hai comes, seemingly disappointed. "It looks like you angered the piloting AI enough to make it override the silence protocols we had put in place. You're lucky it did not kill you, but even so you are of no use to us if you think it wise to use your firearm on a spaceship. Farewell." They turn and leave without another word, and the ship flies off.`
+				decline
 			label lucky??
-			`	You manage to land a lucky hit on what you assume to be the ship's navigation system, because it comes down, seemingly crashing.`
-			`	Except that it crashes right on you, and it seems more like a landing than an actual crash, especially considering the Hai are left mostly untouched, having taken a few leaps back right in time.`
+			`	You manage to land a lucky hit on what you assume to be the ship's navigation system, because it comes down, seemingly crashing. You feel like you hear a robotic voice cry out, but it's hard to tell with all the noise.`
+			`	Unfortunately, the crash is right on top you, and it seems more like a landing than an actual crash, especially considering the Hai are left mostly untouched, having taken a few leaps back right in time.`
```

Comment:
~~are you sure this change is an upgrade?~~

Reply:
```suggestion
			`	Unfortunately, the crash is right on top of you, and it seems more like a landing than an actual crash, especially considering the Hai are left mostly untouched, having taken a few leaps back right in time.`
```